curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-data-structures-by-building-a-shopping-cart/63ee611d478dca0b77f6a393.md

```diff
@@ -0,0 +1,316 @@
+---
+id: 63ee611d478dca0b77f6a393
+title: Step 16
+challengeType: 0
+dashedName: step-16
+---
+
+# --description--
+
+The `this` keyword in JavaScript is used to refer to the current object. Depending on where `this` is used, what it references changes. In the case of a class, it refers to the instance of the object being constructed. You can use the `this` keyword to set the properties of the object being instantiated. Here is an example:
+
+```js
+class Computer {
+  constructor() {
+    this.ram = 16;
+  }
+}
+```
+
+In your constructor, use the `this` keyword to set the `items` property to an empty array. Also set the `total` property to `0`, and the `taxRate` property to `8.25`.
```

Comment:
Comma between Also and set

In your constructor, use the `this` keyword to set the `items` property to an empty array. Also, set the `total` property to `0`, and the `taxRate` property to `8.25`.